libraries/cyclestreets-core/src/main/java/net/cyclestreets/CycleStreetsPreferences.java

```diff
@@ -283,6 +284,23 @@ public static void setPendingUsernamePassword(final String username,
     editor.commit();
   }
 
+  public static void setTempUsernamePassword(final String username,
+                                             final String password,
+                                             final String confirmPassword,
+                                             final String name,
+                                             final String email,
+                                             final boolean pending) {
+    final Editor editor = editor();
+    editor.putString(PREF_USERNAME_KEY, username);
+    editor.putString(PREF_PASSWORD_KEY, password);
+    editor.putString(PREF_CONFIRM_PASSWORD_KEY, confirmPassword);
+    editor.putString(PREF_NAME_KEY, name);
+    editor.putString(PREF_EMAIL_KEY, email);
+    editor.putBoolean(PREF_PENDING_KEY, pending);
```

Comment:
I'm thinking it's better to leave PREF_PENDING_KEY out.  Setting it to true displays message saying that they have already tried  to register an account, which they may not have done yet.